Code/Tools/AssetProcessor/native/unittests/AssetProcessorManagerUnitTests.cpp

```diff
@@ -190,22 +171,44 @@ namespace AssetProcessor
         EXPECT_EQ(jobDetails.m_intermediatePath, intermediateAssetsFolder);
     };
 
-    namespace
+    namespace Internal
```

Comment:
Why move from the anonymous namespace? 

If we are adding this to provide semantic meaning, then maybe give it a better name APUnitTestUtils or similar?

Reply:
In my previous PR, Allen mentioned that O3DE coding practices do not use anonymous namespaces. I can replace it with a more meaningful namespace.